src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/cloud-watch-fallback.mdx

```diff
@@ -0,0 +1,149 @@
+
+It's important to note that function logs are just that, logs that are recorded by the function during its invocation. The extension will not send Lambda runtime logs like `START`, `END`, and `REPORT` lines.
+
+Set [extension environment variables](https://github.com/newrelic/newrelic-lambda-extension/blob/main/config/config.go) on your function:
+
+    * `NEW_RELIC_LAMBDA_EXTENSION_ENABLED`: true (default)
+    * `NEW_RELIC_EXTENSION_SEND_FUNCTION_LOGS`: true
+
+Set your `aws-log-ingestion` function environment variables:
+
+    * `INFRA_ENABLED`: true
+    * `LOGGING_ENABLED`: false (disable log forwarding to avoid duplicates or set the CloudWatch subscription filter pattern to match `NR_LAMBDA_MONITORING` lines only)
+
+If you're using [`serverless-newrelic-lambda-layers`](https://github.com/newrelic/serverless-newrelic-lambda-layers), set:
+
+    ```yaml
+    custom:
+        newRelic:
+            enableExtension: true
+            enableFunctionLogs: true
+            enableIntegration: true
+            cloudWatchFilter: "NR_LAMBDA_MONITORING" (only send payloads)
+    ```
+
+If you're using the [`newrelic-lambda-cli`](https://github.com/newrelic/newrelic-lambda-cli), set: 
+
+    * `newrelic-lambda integrations install --nr-account-id &lt;YOUR_ACCOUNT_ID> --nr-api-key &lt;YOUR_API_KEY>`
+    * `newrelic-lambda layers install --function &lt;name or arn> --nr-account-id &lt;YOUR_NEW_RELIC_ACCOUNT_ID>`
+    * `newrelic-lambda subscriptions install --function &lt;name or arn> --filter-pattern "NR_LAMBDA_MONITORING"`
+
+
+## Always send logs and payloads via CloudWatch [#three]
+
+This option relies completely on CloudWatch and our `aws-log-ingestion` function to ship logs and payloads to New Relic. This option has the extension disabled to avoid potential downtime caused by extension failures.
+
+For this method, the New Relic Lambda extension needs to be disabled completely to avoid encoding the `NR_LAMBDA_MONITORING` line. With the New Relic Lambda extension disabled, all telemetry will go out via CloudWatch + subscription filter + aws-log-ingestion function.
+
+### Configuration [#configuration-3]
+
+After adding a New Relic Lambda layer, the extension is enabled and has log shipping disabled by default. You'll need to disable the extension with an environment variable.
+
+Set [extension environment variables](https://github.com/newrelic/newrelic-lambda-extension/blob/main/config/config.go) on your function as follows:
+
+    * `NEW_RELIC_LAMBDA_EXTENSION_ENABLED`: false
+
+Set your `aws-log-ingestion` function environment variables as follows:
+
+    * `INFRA_ENABLED`: true
+    * `LOGGING_ENABLED`: true
+
+If you're using [`serverless-newrelic-lambda-layers`](https://github.com/newrelic/serverless-newrelic-lambda-layers), set:
+
+    ```yaml
+    custom:
+    newRelic:
+        enableExtension: false
+        enableIntegration: true
```

Comment:
The integration option should only be included if the function is being deployed to an AWS account that does not yet have an integration. (Then, once the integration is set up, this option should be removed from the yaml of the deployed function.)